### PR TITLE
Added renewal and pre renewal warps to prontera to match Royal Banquet Episode.

### PR DESCRIPTION
**Pull request by SinjiPrasetio**
- Added pre-renewal and renewal option for prontera warps, to match Royal Banquet Episode.
- Added commented morroc warps. (destroyed morroc which is disabled in latest episode).

### CODE REVIEW

**Comment by sagunkho** [comment]
There was an error detected in the run - https://github.com/horizonxyz/horizon/actions/runs/7307132963/job/19913983690?pr=145
Screenshot 2023-12-23 170246
Could this be because "npcs/npcs.lua" was not found? Since there seems to be no such file in the folder.

File: scripts/include.lua [hunk 1]
```diff
@@ -146,6 +147,10 @@ scripts = {
 	"npcs/re/warps/other/dimensional_gap.lua",
 	"npcs/re/warps/other/arena.lua",
 
+	-- NPC
+	"npcs/npcs.lua",
```
Comment (sagunkho) [inline scripts/include.lua:151]: Related: https://github.com/horizonxyz/horizon/pull/145#issuecomment-1868274806

**Review by sagunkho — CHANGES_REQUESTED** [review]
Reviewed, requesting changes.

**Comment by SinjiPrasetio** [comment]
> Reviewed, requesting changes.

Fixed, remove the unused lua files.

**Review by sagunkho — APPROVED** [review]
Commit changed as per request. PR Approved.

**Pull request merged**